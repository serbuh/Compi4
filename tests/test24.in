// Simple foreach statement
K x = 100 K;
K y = 200 K;
C z = 0 C;
int i = 8;

foreach(i K) + 10;
print x;
print y;
print z;
