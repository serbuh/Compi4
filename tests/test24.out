180
280
0
